providers/gitlab_test.go

```diff
@@ -108,6 +127,46 @@ func TestGitLabProviderUsername(t *testing.T) {
 	assert.Equal(t, "FooBar", username)
 }
 
+func TestGitlabProviderProject(t *testing.T) {
+	b := testGitLabBackend()
+	defer b.Close()
+
+	bURL, _ := url.Parse(b.URL)
```

Comment:
I see the older tests do this too -- but lets not ignore the errors.

Can we add a `assert.NoError(t, err)` after this?